readme.md

```diff
@@ -62,6 +62,7 @@
 - [Scrims](https://github.com/brettgullan/tailwindcss-scrims) - Adds scrims utilities.
 - [Colors to CSS Variables](https://github.com/n1kk/tailwind-color-vars) - Exports color configuration to CSS Custom Properties.
 - [CSS Variables](https://github.com/omarkhatibco/tailwind-css-variables) - Exports configuration to CSS Custom Properties.
+- [CSS Logical Properties](https://github.com/omarkhatibco/tailwind-css-logical-properties) - Generate classnames for CSS Logical Properties for margin, padding, border-width, border-raduis, text-align, float & writing-mode.
```

Comment:
This needs to be at the bottom of the list. Please read [item 3 on the guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/contributing.md).

Reply:
hi Josh (@joshmanders)

I'm sorry for this mistake. it fixed now.